aah/list.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Jeevanandam M. (https://github.com/jeevatkm)
+// go-aah/tools source code and usage is governed by a MIT style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"aahframework.org/essentials.v0"
+)
+
+const aahProjectIdentifier = "aah.project"
+
+var (
+	listCmd = &command{
+		Name:      "list",
+		UsageLine: "aah list",
+		Short:     "List all aah projects in GOPATH",
+		Long: `aah's list command allows you to view all projects that are
+  making use of aah in your GOPATH`,
+		Run: listRun,
+	}
+)
+
+func listRun(args []string) {
+
+	gopath, err := ess.GoPath()
+
+	if err != nil {
+		fmt.Println("Your GOPATH could not be determined .... exiting")
+		os.Exit(1)
+	}
+
+	var count int
+
+	filepath.Walk(gopath, func(path string, info os.FileInfo, err error) error {
+
+		if err != nil {
+			return err
+		}
+
+		if info.IsDir() {
+			return nil
```

Comment:
If `SkipDir` is returned, the entire content of that directory would be skipped (https://golang.org/pkg/path/filepath/#WalkFunc).. And since the file path is being inspected for `aah.project`, the command wouldn't work..